run-tests.sh

```diff
@@ -25,7 +25,7 @@
 
 pep257 invenio_accounts && \
 check-manifest --ignore ".travis-*" && \
-#isort -rc -c -df **/*.py && \
+# isort -rc -c -df **/*.py && \
```

Comment:
if you are touching this please uncomment it and fix the `.editorconfig [isort]`